### DISCUSSION TITLE
Have attached Mike Chappel's book.

### DISCUSSION THREAD

**Discussion by testexam238712** [AyemunHossain/ISC2-CC-Dump-Questions-Study-Material#6] [General]
Have attached Mike Chappel's book. 
[CC Certified in Cybersecurity Study Guide - Mike Chapple (2024).pdf](https://github.com/user-attachments/files/18252767/CC.Certified.in.Cybersecurity.Study.Guide.-.Mike.Chapple.2024.pdf)

**Comment by AyemunHossain** [comment]
In : [ISC2-CC-Dump-Questions-Study-Material](https://github.com/AyemunHossain/ISC2-CC-Dump-Questions-Study-Material) we have enough for now.

**Reply by testexam238712** [reply]
Thank you. Is the 205 dump questions true to the exam. Thanks.

**Reply by Logic1252** [reply]
Unfortunately not. Its great study material, but not one question was on the exam.

**Reply by testexam238712** [reply]
> Unfortunately not. Its great study material, but not one question was on the exam.

Same experience. I passed tho so was good practice. Did you pass also?

**Reply by AyemunHossain** [reply]
Not just me, hundred of peoples are passing with this materials.
